<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvalForm</class>
 <widget class="QWidget" name="EvalForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evaluation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>

   <!-- 使用 GroupBox 包装第一部分 -->
   <item>
    <widget class="QGroupBox" name="groupLabelme">
     <property name="title">
      <string>Convert LabelMe JSON</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_json">
        <item>
         <widget class="QLabel" name="labelTrainJson">
          <property name="text">
           <string>JSON path：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineTrainJson"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectJson">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelPolicy">
        <property name="text">
         <string>Label Rendering Policy:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineLabelFilter">
        <property name="placeholderText">
         <string>Enter labels (e.g., cat;dog;person)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linePolicy">
        <property name="placeholderText">
         <string>Enter policies (e.g., Closure;Follow;Follow)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tablePolicy">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <row/>
        <column>
         <property name="text">
          <string>Label</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Policy</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLabelme2Mask">
        <property name="text">
         <string>Transfer LabelMe JSON into Mask Images</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- 使用 GroupBox 包装第二部分 -->
   <item>
    <widget class="QGroupBox" name="groupEval">
     <property name="title">
      <string>Segmentation Evaluation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_mask1">
        <item>
         <widget class="QLabel" name="labelMask1">
          <property name="text">
           <string>Mask File Path 1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineMask1"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectMask1">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_mask2">
        <item>
         <widget class="QLabel" name="labelMask2">
          <property name="text">
           <string>Mask File Path 2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineMask2"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectMask2">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnEval">
        <property name="text">
         <string>Make Evaluation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>